lib/fog/ovirt/compute/v4.rb

```diff
@@ -60,6 +60,11 @@ def ovirt_attrs(obj)
                 next
               end
 
+              if key == :storage_domain && obj.is_a?(OvirtSDK4::Disk) && obj.storage_domains.size == 1
```

Comment:
This is because the storage domain did not show up in foreman so cloning would send the first in the list of possible storage domains.